src/main/java/uk/gov/hmcts/sidam/IdObject.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.sidam;
+
+public class IdObject {
+
+	private long id;
```

Comment:
spaces does not look stylistically correct. pls double-check.

Reply:
removed